src/components/calcite-fab/calcite-fab.tsx

```diff
@@ -0,0 +1,106 @@
+import { Component, Element, Host, Method, Prop, h } from "@stencil/core";
+import { CalciteTheme } from "../interfaces";
+import { CSS } from "./resources";
+import { focusElement, getElementDir } from "../utils/dom";
+import { plus24F } from "@esri/calcite-ui-icons"; // todo: will remove once icon string supported in `calcite-button`
```

Comment:
Unless this is happening before this is merged we should probably have an issue for this.

Reply:
@asangma do we need the FAB now? or can we wait on the next calcite-components release with the icon changes?

Reply:
They released. Made the changes.